src/components/table/style-utilities.jsx

```diff
@@ -4,15 +4,15 @@ const sizes = () => ({
   },
 
   '&.sm': {
-    fontSize: '14px',
+    fontSize: '12px',
```

Comment:
Is px needed here or can we leverage JSS here?

Reply:
ask Christian why he didn't change it, but should work as numbers